news/738a71b0-98f9-4e1f-a541-af95fb990af9.bugfix.rst

```diff
@@ -0,0 +1 @@
+Harmonize type signature of ``VersionControl.get_remote_url()`` subclasses.
```

Comment:
This would be trivial, since this doesn't change user-facing behaviour.

Reply:
Works for me. Changed to trivial in the latest revision.